### ISSUE TITLE
Error on `dfm init`

### ISSUE THREAD

**Issue by FedericoStra** [chasinglogic/dfm#55]
# Error

I just installed `dfm`. The folder `~/.config/dfm` does not exist.

```
[:~] $ dfm --version
dfm version 8.4.2
[:~] $ dfm init default
fatal: not a git repository (or any of the parent directories): .git
Traceback (most recent call last):
  File "/usr/local/bin/dfm", line 8, in <module>
    sys.exit(main())
  File "/usr/local/lib/python3.10/site-packages/dfm/cli/__init__.py", line 95, in main
    command_mod.run(docopt(command_mod.__doc__, argv=argv))
  File "/usr/local/lib/python3.10/site-packages/dfm/cli/init_cmd.py", line 25, in run
    Profile.new(new_profile_dir)
  File "/usr/local/lib/python3.10/site-packages/dfm/profile.py", line 270, in new
    profile = cls.default(where)
  File "/usr/local/lib/python3.10/site-packages/dfm/profile.py", line 179, in default
    return cls(df_repo, link_manager, hooks, **extras)
  File "/usr/local/lib/python3.10/site-packages/dfm/profile.py", line 88, in __init__
    self.repo = self.df_repo.get_remote()
  File "/usr/local/lib/python3.10/site-packages/dfm/repo.py", line 45, in get_remote
    return subprocess.check_output(
  File "/usr/lib64/python3.10/subprocess.py", line 420, in check_output
    return run(*popenargs, stdout=PIPE, timeout=timeout, check=True,
  File "/usr/lib64/python3.10/subprocess.py", line 524, in run
    raise CalledProcessError(retcode, process.args,
subprocess.CalledProcessError: Command '['git', 'remote', 'get-url', 'origin']' returned non-zero exit status 128.
```

Apparently `dfm` is trying to get the URL of the remote `origin`, which clearly I haven't had the time to define.

What am I doing wrong? Is this the wrong procedure to start using `dfm` with a new profile for the first time?

**Comment by chasinglogic** [comment]
Hey @FedericoStra this is not the wrong process. Looks like a legit bug.

Let me see if I can replicate this, are you using Linux or MacOS? (Also if using Mac is this with Homebrew Python)

**Comment by FedericoStra** [comment]
```bash
[:~] $ uname -srom
Linux 5.15.16-200.fc35.x86_64 x86_64 GNU/Linux
[:~] $ distro
Name: Fedora Linux 35 (Workstation Edition)
Version: 35
Codename: 
[:~] $ dfm --version
dfm version 8.4.2
[:~] $ python --version
Python 3.10.2
[:~] $ git --version
git version 2.34.1
```

**Comment by chasinglogic** [comment]
Alrighty I'll see if I can replicate this tonight / tomorrow. I've got a Fedora box sitting around. Thanks for the info and bug report 👍

**Comment by chasinglogic** [comment]
I've pushed a fix, can you try upgrading to `8.4.3` with `pip3 install -U dfm` and let me know if this resolves your issue.

**Comment by FedericoStra** [comment]
Works like a charm :rainbow:
```text
[:~] $ dfm init default
Initialized empty Git repository in /home/.../.config/dfm/profiles/default/.git/
```

I'll start playing with this interesting tool as soon as I find the time! :soon:
